app/views/proposals/_proposal_list.html.erb

```diff
@@ -17,11 +17,20 @@
   <%- end %>
   <%- proposals_list.each do |proposal| %>
     <tr>
-      <td class="sixth"><a href="/carts/<%= proposal.cart.id %>"><%= proposal.cart.id %></a></td>
-      <td class="first"><a href="/carts/<%= proposal.cart.id %>"><%= proposal.name %></a></td>
+      <td class="sixth"><a href="<%= proposal_url(proposal) %>"><%= proposal.public_identifier %></a></td>
+      <td class="first"><a href="<%= proposal_url(proposal) %>"><%= proposal.name %></a></td>
       <td class="sixth"><%= number_to_currency(proposal.total_price) %></td>
       <td class="sixth">
-        <p><%= display_status(proposal.cart, current_user) %></p>
+        <% approvers = proposal.currently_awaiting_approvers %>
+        <% if proposal.pending? && approvers.include?(current_user) %>
+          <%# This isn't used anywhere %>
```

Comment:
The condition would be true if the user were an approver on an outstanding approval, but we only list carts by the requester. As the requester shouldn't be an approver on his/her own cart, I don't think this text is ever shown.